app/views/recipes/_table.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="flex flex-col">
+  <div class="overflow-x-auto sm:-mx-6 lg:-mx-8">
+    <div class="py-2 inline-block min-w-full sm:px-6 lg:px-8">
+      <div class="overflow-hidden">
+        <table class="min-w-full">
+          <thead class="border-b">
+            <tr>
+              <th scope="col" class="text-sm font-medium text-gray-900 px-6 py-4 text-left">
+                #
+              </th>
+              <% headers.each do |t|%>
+                <th scope="col" class="text-sm font-medium text-gray-900 px-6 py-4 text-left">
+                <%= t %>
+              </th>
+              <% end %>
+            </tr>
+          </thead>
+          <tbody>
+            <% foods.includes(:recipe_foods).each_with_index do |food, i| %>
+              <tr class="border-b">
+                <td class="px-6 py-4 whitespace-nowrap text-sm font-medium text-gray-900"><%= i + 1 %></td>
+                <td class="text-sm text-gray-900 font-light px-6 py-4 whitespace-nowrap">
+                  <%= food.name %>
+                </td>
+                <td class="text-sm text-gray-900 font-light px-6 py-4 whitespace-nowrap">
+                  <%= food.recipe_foods[i].quantity %>
+                </td>
+                <td class="text-sm text-gray-900 font-light px-6 py-4 whitespace-nowrap">
+                 <strong>$</strong><%= food.price * food.recipe_foods[i].quantity.to_i %>
+                </td>
+                <td class="flex gap-2 text-sm text-gray-900 font-light px-6 py-4 whitespace-nowrap">
+                <%= form_with(model: RecipeFood, url:  edit_user_recipe_recipe_food_path(current_user, food.id, food.recipe_foods[i].id), method: :GET) do |form| %>
+                  <%= form.submit 'Modify', class: "underline text-blue-500 pointer-cursor" %>
+                 <% end %>
+                 <%= form_with(model: RecipeFood, url:  user_recipe_recipe_food_path(current_user, food.id, food.recipe_foods[i].id), method: :delete) do |form| %>
+                  <%= form.submit 'Remove', class: "underline text-blue-500" %>
+                 <% end %>
```

Comment:
- [ ] As the previous reviewer mentioned, There is an error raised here when I try to open the recipe details(show) page. So please fix that.

Reply:
This link is working properly on my end and a friend's machine as well. I am surprised it is not working for you and @youmari. I would have loved to discuss this with you both over a call but seems you are both not reachable, so I am going to submit it again and hope the next reviewer gets to talk with me if they have any issues with this route.

Reply:
Hi @aceDavon,
You may not have this error because you already have data on your database, So try resetting your database `rails db:reset` and run the rails server and test your app. You will see the problem we mentioned.

Reply:
Thanks, @Omar-Muhamad for your response, I have dropped my database and migrated afresh and still do not have this error, I have also just run the rails DB:reset you suggested and still do not have this error. Please what branch of my project did you check? I am presently requesting a review from the dev branch against the main branch. Can I come on a call with you to see this error on your end?

Reply:
![Screenshot from 2022-12-16 15-45-31](https://user-images.githubusercontent.com/57240208/208123586-5dbacb38-ddc3-47a8-ae41-f93c31f863d5.png)

Here is a screenshot of my recipes show page

Reply:
I tested your app again and I can see what's happening now:
When I tried to create a recipe using only one food everything worked but when I create a recipe using multiple foods now this problem appeared.

Reply:
Ok, I've noted your recommendations and will probably remove the food addition from the add recipe form. Once you recipe is created, you can then proceed to add the foods for the recipe. Thanks for your time tho.